packages/build/tests/error/snapshots/tests.js.md

```diff
@@ -2022,40 +2006,40 @@ Generated by [AVA](https://ava.li).
        - onEnd␊
     ␊
       Plugin details␊
-      Package:        /file/path␊
+      Package:        plugin.js␊
       Version:        1.0.0␊
       Repository:     git+https://github.com/netlify/build.git␊
       Report issues:  https://github.com/netlify/build/issues␊
     ␊
       Error location␊
-      While loading "/file/path" from netlify.toml␊
+      While loading "plugin.js" from netlify.toml␊
     STACK TRACE␊
     ␊
       Resolved config␊
       plugins:␊
         - inputs: {}␊
           origin: config␊
-          package: /file/path␊
+          package: plugin.js␊
     ␊
     Error monitoring payload:␊
     {␊
       "errorClass": "pluginValidation",␊
-      "errorMessage": "Invalid event 'invalid'/file/path use a valid event name. One of:/n - onPreBuild/n - onBuild/n - onPostBuild/n - onSuccess/n - onError/n - onEnd",␊
```

Comment:
These extra `Please`'s were maybe being caught by the sting replacement before? 🤔

Reply:
I believe so!